codice/ddf
# [2.13.x] DDF-4311 Remove gazetteer UI component, and allow URLs in the CLI

Backports #4003 

What does this PR do?
Removes the offline gazetteer from the UI, adds url functionality to the CLI, and updates the gazetteer documentation and CLI help message to reflect the change.

The UI component was removed because one of the endpoints used by the UI no longer exists. 

@emmberk has added a fix for a related issue to this PR. The issue prevented loading geocode files.

Who is reviewing it?
@mdang8 @kcover @emmberk @peterhuffer 

Ask 2 committers to review/merge the PR and tag them here.
@clockard @andrewkfiedler 

How should this be tested?
Try to add geocodes through the command line by using a link.
gazetteer:update linkToResource Links can be found here

In the admin UI, verify the GeoNames Configuration tab under the Spatial app, is no longer there.

What are the relevant tickets?
[DDF-4344](https://codice.atlassian.net/browse/DDF-4344)

Screenshots
Before:
![](https://user-images.githubusercontent.com/12192559/48137906-136ea400-e271-11e8-9a76-df9f6feb1ebc.png)

After:
![](https://user-images.githubusercontent.com/12192559/48137914-16699480-e271-11e8-8aa4-c2c3d8f59ba1.png)

Notes on Review Process
Please see Notes on Review Process for further guidance on requirements for merging and abbreviated reviews.

Review Comment Legend:
✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist.
❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.
